test/peaks/yearn/YVaultController.test.js

```diff
@@ -0,0 +1,129 @@
+const assert = require('assert')
+const utils = require('../../utils.js')
+
+const toWei = web3.utils.toWei
+const fromWei = web3.utils.fromWei
+const toBN = web3.utils.BN
+const MAX = web3.utils.toTwosComplement(-1);
+const n_coins = 4
+
+/*
+Problems need to be solved
+
+1) How to populate controller with x amount of yCRV
+2) { from: this.yVaultPeak.address } => Error: sender not recognised
```

Comment:
You cannot, test `onlyPeak` flows via making calls through yvaultPeak. Tests for the controller don't have to be fully isolated.